api_notificationhub_messages.yaml

```
@@ -0,0 +1,101 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy API
+  description: Mobile and web proxy API gateway.
+host: localhost
+basePath: /pagopa/api/v1
+schemes:
+  - https
+paths:
+definitions:
+  Platform:
+    $ref: 'https://raw.githubusercontent.com/pagopa/io-backend/v5.19.1/api_backend.yaml#/definitions/Platform'
+  NotificationHubMessageKind:
+    type: string
+    enum:
+      - "NotifyKind"
+      - "CreateOrUpdateKind"
+      - "DeleteKind"
```

```suggestion
      - "Notify"
      - "CreateOrUpdateInstallation"
      - "DeleteInstallation"
```

cc @AleDore use the enum from here in io-functions-app

---

Fixed in c925ce9